src/routes/+page.server.ts

```diff
@@ -1,69 +1,65 @@
+import type { SupabaseClient } from '@supabase/supabase-js';
 import type { PageServerLoad } from './admin/$types';
-import { error, fail } from '@sveltejs/kit';
+import type { Database } from '$lib/types/database.generated.types';
+import type { MatchStatsQuery } from '$lib/types/newTypes';
 
 export const load: PageServerLoad = async ({ locals: { supabase }, parent }) => {
 	let { session, season } = await parent();
 
 	if (session) {
 		if (!season) return {};
 
-		const { data: teams, error: teamError } = await supabase
-			.from('teams')
-			.select(
-				`
+		return {
+			season,
+			teams: getTeamsForSeason(season.id, supabase),
+			matches: getMatchesForSeason(season.id, supabase),
+			lazy: {
+				fantasyTeams: getFantasyTeamsForSeason(season.id, supabase)
+			}
+		};
```

Comment:
testing the lazy loading stuff which might be useful for skeleton loaders. putting skeleton loaders aside because i'd rather get stuff up 2 snuff before we make things pretty

Reply:
This is also why I'm not handling the errors on the server, but rather in the await/then/catch block. Not super pretty because this PR ended up being a few different things at once